homeassistant/components/mystrom/const.py

```diff
@@ -2,3 +2,5 @@
 DOMAIN = "mystrom"
 DEFAULT_NAME = "myStrom"
 MANUFACTURER = "myStrom"
+SENSOR_TYPE_CONSUMPTION = "consumption"
+SENSOR_TYPE_TEMPERATURE = "temperature"
```

Comment:
```suggestion
```

Reply:
why do you think the usage of constants is a bad thing here?

Reply:
I think because it's only used in one place, so there's not a real benefit to make it a constant at this point.